File: client/src/classes/inventory/waifu_consumable.ts
import type WaifuModifier from "./modifiers/waifu_modifier";
import Item from "./item";
import ItemType from "./item_type";

export default class WaifuConsumable extends Item {
    public modifiers : WaifuModifier[] = [];
    public type : ItemType = ItemType.WaifuConsumable
}

File: server/UserData/Item/waifu_consumable.cs
using Nanina.UserData.ModifierData;

namespace Nanina.UserData.ItemData
{
    public class WaifuConsumable : Item
    {
        public new ItemType type {get; set;} = ItemType.WaifuConsumable;
        public WaifuModifier[] modifiers {get; set;}
    }

}